pants-plugins/examples/bash/bash_setup.py

```diff
@@ -19,9 +19,8 @@
 
 # See https://www.pantsbuild.org/v2.0/docs/rules-api-subsystems.
 class BashSetup(Subsystem):
-    """Common setup for the Bash plugin."""
-
     options_scope = "bash-setup"
+    help = """Common setup for the Bash plugin."""
```

Comment:
No need for triple quotes anymore in these help messages

Reply:
This is how `fmt` left them, so I'm happy with it.